spyder/plugins/editor/widgets/editor.py

```diff
@@ -2820,11 +2820,12 @@ def __init__(self, parent, plugin, menu_actions, show_fullpath,
         self.find_widget = FindReplace(self, enable_replace=True)
         self.plugin.register_widget_shortcuts(self.find_widget)
         self.find_widget.hide()
-        self.outlineexplorer = OutlineExplorerWidget(self,
-                                            show_fullpath=show_fullpath,
-                                            show_all_files=show_all_files,
-                                            group_cells=group_cells,
-                                            show_comments=show_comments)
+        self.outlineexplorer = OutlineExplorerWidget(
+                self,

```

Comment:
I think a 4-space indentation should be used here instead of 8 (even though it is what Spyder is proposing by default...). Also, this could be written on less lines. Something like:

```python
self.outlineexplorer = OutlineExplorerWidget(
    self, show_fullpath=show_fullpath, show_all_files=show_all_files,
    group_cells=group_cells, show_comments=show_comments)
```

@ccordoba12 Am I ok with this so that I do not repeat wrong code style in my own PR.

Reply:
I think it's fine as it is. Besides, IMHO it's more readable.

Reply:
Ok thanks for the feedback about keeping one line per argument, but what about the indentation? 

I've been asked in another PR to use 4 spaces instead of 8 as per some pep rules. Are we following any convention in Spyder for this?

Reply:
I think 8 spaces is nice to differentiate it as a line continuation and not a 'normal' code block, but it really doesn't matter.

Reply:
>but it really doesn't matter.

I know. I do not care much either, but I'm trying to follow the "official" codestyle that is used in the spyder codebase as much as possible. I'm asking so that I can save revision time in my future PR.

Reply:
> but I'm trying to follow the "official" codestyle

Wed don't really have one. In this case I think using 8 spaces is fine too.

Reply:
> Wed don't really have one. In this case I think using 8 spaces is fine too.
Alright, thank you Carlos. This is good to know.